.circleci/config.yml

```diff
@@ -148,7 +148,7 @@ jobs:
       - restore_cache: { key: 'gradle-wrapper-v2-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}' }
       - restore_cache: { key: 'publish-gradle-cache-v2-{{ checksum "versions.props" }}-{{ checksum "build.gradle" }}' }
       - deploy:
-          command: ./gradlew --parallel --stacktrace --continue publish
+          command: ./gradlew --parallel --stacktrace --continue publish publishToSonatype closeAndReleaseSonatypeStagingRepository
```

Comment:
Should we wire up these tasks to publish to keep this template more simple?

Reply:
Might as well, I don't see any harm in doing it, considering this just ran `publish` before.